src/main/java/uk/gov/register/filters/ContentSecurityPolicyFilter.java

```diff
@@ -12,6 +12,6 @@
 public class ContentSecurityPolicyFilter implements ContainerResponseFilter {
     @Override
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
-        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com");
+        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com; font-src data:;");
```

Comment:
[This](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/font-src) seems to suggest adding `font-src:data` is a bad idea (I'm happy to be convinced otherwise though):

> data: Allows data: URIs to be used as a content source. This is insecure; an attacker can also inject arbitrary data: URIs. Use this sparingly and definitely not for scripts.

Can we use hashes of the fonts we expect instead?